default.py

```diff
@@ -228,15 +213,13 @@ def display_seasons_weeks(self):
     def display_shows_episodes(self, show_name, season):
         """Show episodes for a given season/show"""
         self.games_items = []
-        items = gp.get_shows_episodes(show_name, season)
+        episodes = gp.get_shows_episodes(show_name, season)
 
-        for i in items['modules']['archive']['content']:
+        for episode in episodes:
             try:
                 listitem = xbmcgui.ListItem('[B]%s[/B]' % show_name)
-                listitem.setProperty('game_info', i['title'])
-                #listitem.setProperty('away_thumb', gp.image_url + i['image'])
-                #listitem.setProperty('url', i['publishPoint'])
```

Comment:
These were intended to be restored by #295. Just in case you're feeling ambitious. ;-)

If not, it makes sense to remove them, as you have, until those thumbnails are restored.